packages/@aws-cdk/core/test/stack.test.ts

```diff
@@ -867,6 +867,43 @@ describe('stack', () => {
     expect(stack.resolve(stack.region)).toEqual('es-norst-1');
   });
 
+  describe('@aws-cdk/core:enablePartitionLiterals', () => {
```

Comment:
Please make this descriptive instead of listing the flag. Better readability that way.

Reply:
This is done in the latest update; thanks